app.json

```diff
@@ -26,7 +26,8 @@
         "NSPhotoLibraryUsageDescription": "Diese App nutzt die Medienbibliothek für die Auswahl eines Profilbildes.",
         "NSLocationWhenInUseUsageDescription": "Diese App nutzt die Standortbestimmung um Ihre aktuelle Position auf der Karte darzustellen.",
         "CFBundleLocalizations": ["de"],
-        "CFBundleDevelopmentRegion": "de_DE"
+        "CFBundleDevelopmentRegion": "de_DE",
+        "LSApplicationQueriesSchemes": ["whatsapp"]
```

Comment:
what is the source of this? where do you got it from? what is the default value? do we need to add more here to make other things still work?

Reply:
It's mentioned in the expo documents. If we want to add more redirects to the application then we need to add them here.

https://docs.expo.dev/guides/linking/#custom-url-schemes